### PR TITLE
fix(website): space bg fadeIn

### PR DESCRIPTION
**Pull request by Shipow**

### CODE REVIEW

**Comment by vvo** [comment]
LGTM

**Comment by vvo** [comment]
I'll let another more documentation experienced reviewer have a look, just for the sake of it. It all looks good of course

**Comment by redox** [comment]
:+1:

File: docs/css/_documentation.sass [hunk 1]
```diff
@@ -208,6 +207,7 @@
       color: $brand-primary
       background-color: #0A1724
       border: none
+      transition: all .2s
```
Comment (bobylito) [inline docs/css/_documentation.sass]: I wouldn't do that...

Comment (Shipow) [inline docs/css/_documentation.sass]: sure, will move to opacity only.

Comment (bobylito) [inline docs/css/_documentation.sass]: I can do it :) (working on it right now :))

**Pull request merged**